src/sql-parser/src/parser.rs

```diff
@@ -2267,7 +2286,7 @@ impl<'a> Parser<'a> {
                     let connection = self.parse_raw_name()?;
                     let options = if self.consume_token(&Token::LParen) {
                         let options = self.parse_comma_separated(
-                            Parser::parse_kafka_broker_aws_private_link_option,
+                            Parser::parse_kafka_broker_aws_privatelink_option,
```

Comment:
This is just fixing an inconsistency... separating privatelink to private_link.

Reply:
thank you!